state/etcd/etcd.go

```diff
@@ -0,0 +1,197 @@
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+
+package etcd
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/dapr/components-contrib/state"
+	jsoniter "github.com/json-iterator/go"
+	"go.etcd.io/etcd/clientv3"
+	"google.golang.org/grpc"
+)
+
+const defaultOperationTimeout = time.Duration(10 * time.Second)
+const defaultSeparator = ","
+
+var errMissingEndpoints = errors.New("Endpoints are required")
+var errInvalidDialTimeout = errors.New("DialTimeout is invalid")
+
+// StateStore is a Etcd state store
+type StateStore struct {
```

Comment:
Hey, can you rename this to `ETCD`? I know in Redis we have it as StateStore, but that's going to change :)

Reply:
Just `ETCD` or `EtcdStateStore` ?

Reply:
Just ETCD. Thanks!

Reply:
👍